go.mod

```diff
@@ -32,6 +32,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.19.0
 	github.com/prometheus/client_model v0.6.0
+	github.com/prometheus/procfs v0.12.0
```

Comment:
I wondered if we could avoid adding another explicit dependency for Tetragon, maybe for the little usage in `procsyms.go` we can do it "by hand" as we do so far.